src/System.DirectoryServices.AccountManagement/tests/PrincipalTest.cs

```diff
@@ -111,30 +89,28 @@ public void AddExistingPrincipal()
             }
             catch (PrincipalExistsException)
             {
-                exceptionThrown = true;
             }
 
             // validate that we correctly throw an exception when trying to add an existing principal
-            Assert.IsTrue(exceptionThrown);
+            //Assert.IsTrue(exceptionThrown);
```

Comment:
At the time of this pull request I hadn't had time yet to familiarize myself with Nunit -> Xunit conversions.    I went through and fixed these up, along with a few others.